compiler/semtempl.nim

```diff
@@ -560,7 +560,7 @@ proc semTemplateDef(c: PContext, n: PNode): PNode =
     s = semIdentVis(c, skTemplate, n.sons[0], {})
 
   if s.owner != nil and sfSystemModule in s.owner.flags and
-      s.name.s in ["!=", ">=", ">", "incl", "excl", "in", "notin", "isnot"]:
+      s.name.s in ["!=", ">=", ">", "incl", "excl", "in", "notin", "isnot", "..<"]:
```

Comment:
Yay, unrelated commits.

Reply:
Well it is not unrelated, ``..<`` is used a lot in the code and I needed this change to debug what I was working on. I simply don't want to continue debugging further problems in this branch without this fix being active, it is likely to be important.

Reply:
ok.